helmfile.d/stacks/tekton.yaml

```diff
@@ -0,0 +1,30 @@
+---
+templates:
+  tekton-pipelines-podsecuritypolicy:
+    inherit: [ template: podsecuritypolicies ]
+    installed: {{ .Values | get "tektonPipelines.enabled" false }}
+    condition: ck8sManagementCluster.enabled
+    namespace: tekton-pipelines
+    name: tekton-pipelines-podsecuritypolicy
```

Comment:
```suggestion
```
Currently we identify podsecuritypolicy releases by `<namespace>/podsecuritypolicy`, which is set by default in the psp template.